styles/styles.css

```diff
@@ -0,0 +1,115 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans' rel="stylesheet");
+
+.container {
+  font-family: 'Noto Sans', sans-serif;
+  text-align: center;
+  width: 100%;
+  }
+
+ul {
+  list-style-type: none;
+}
+
+nav ul {
+  text-align: left;
+  font-weight: bold;
+  display: grid;
+  grid-template-columns: 60% 20% 20%;
+}
+
+.top {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.top a {
+  margin: 3em;
+  border: solid #9ea1a5 thin;
+  border-radius: 10px;
+  text-decoration: none;
+  color: black;
+  padding: 10px;
+  font-size: 1em;
+}
+
+.quote {
+  line-height: 2em;
+  font-family: serif;
+  font-style: italic;
+  font-size: 25px;
+  padding: 125px, 125px;
+}
+
+.quote p {
+  text-align: right;
+  padding-right: 10%;
+}
+
+.mission ul {
```

Comment:
I appreciate your excellent use of grid here.  Your relative measurements make sure each list item takes up the same amount of space on the page.  Great job!